modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -433,6 +444,10 @@
 	}
 }
 
+.col-md-4 .calendar-portlet .popover.scheduler-event-recorder-popover {
+	max-width: 14.5rem;
```

Comment:
About this unit of measurement for `max-width`, why did you have to use the `rem`? It is relative to the parent's width and if the parent's width changes, that `max-width` will also be changed. Knowing this, wouldn't it be the case to set the `max-width` to `pixel`?

Reply:
Yes, I agree with you. I just repeated the default value that was defined in the popover class, but I will put it in pixels, thank you :)